channeldb/forwarding_policy.go

```diff
@@ -0,0 +1,143 @@
+package channeldb
+
+import (
+	"github.com/lightningnetwork/lnd/kvdb"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+var (
+	// initialChannelForwardingPolicyBucket is the database bucket used to
+	// store the forwarding policy for each permanent channel that is
+	// currently in the process of being opened.
+	initialChannelForwardingPolicyBucket = []byte(
+		"initialChannelFwdingPolicy",
+	)
+)
+
+// InitialForwardingPolicy describes the set of initial constraints that a given
+// ChannelLink is to adhere to when forwarding HTLCs directly after it is first
+// created. This is identical to the htlcswitch.ForwardingPolicy struct, but is
+// used to avoid an import cycle.
+type InitialForwardingPolicy struct {
```

Comment:
Makes me think we should add a `TODO` to replace `htlcswitch.ForwardingPolicy` with this one.

Reply:
That's a good idea, I'll move it there.